jenkins/runGobenchPerformance.fish

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env fish
+set -xg simple (pwd)/performance
+set -xg gobenchdir (pwd)/gobench
+set -xg date (date +%Y%m%d)
+set -xg datetime (date +%Y%m%d%H%M)
+
+if test -z "$ARANGODB_TEST_CONFIG"
+  set -xg ARANGODB_TEST_CONFIG run-small-edges.js
```

Comment:
I think this is not correct here.
That config is only relevant for simplePerfTest